.gitignore

```diff
@@ -31,4 +31,6 @@ Session.vim
 
 e2e-tests
 
-coverage.txt
\ No newline at end of file
+coverage.txt
+
+rootfs/etc/nginx/lua/vendor/resty/**
```

Comment:
Any reason why you aren't pushing this to the repository?

Reply:
If the plan is to eventually remove pulling the dependencies from the build.sh file, then these need to be pushed. I will remove this.